binder/requirements.txt

```diff
@@ -0,0 +1,2 @@
+wasabi>=0.2.1,<1.1.0
+allennlp
```

Comment:
@matt-gardner Do you think we should install from the latest master instead, e.g., `git+https://github.com/allenai/allennlp.git`? Is there a good commit / tag that we should anchor this to?

Reply:
For now, yeah, latest master is probably best.  We don't have a specific tag, and won't until we release 1.0.  I'd probably just grab the SHA from the latest commit and use that, updating it when there's something that we want to incorporate here.